cmake/projects/libxml2/schemes/url_sha1_libxml2_msvc.cmake.in

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2016, Ruslan Baratov
+# Copyright (c) 2017, Sacha Refshauge
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+project(Hunter)
+
+include(ExternalProject) # ExternalProject_Add
+
+# Scheme for download and install LibXML2 library for Windows
+
+list(APPEND CMAKE_MODULE_PATH "@HUNTER_SELF@/cmake/modules")
+include(hunter_libxml2_install_tools)
+include(hunter_fatal_error)
+include(hunter_internal_error)
+include(hunter_status_debug)
+include(hunter_test_string_not_empty)
+
+hunter_status_debug("Scheme: url_sha1_libxml2_msvc")
+
+# Check preconditions
+hunter_test_string_not_empty("@HUNTER_SELF@")
+hunter_test_string_not_empty("@HUNTER_EP_NAME@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_URL@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_SHA1@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_DOWNLOAD_DIR@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_SOURCE_DIR@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_INSTALL_PREFIX@")
+hunter_test_string_not_empty("@HUNTER_MSVC_ARCH@")
+hunter_test_string_not_empty("@HUNTER_MSVC_VCVARSALL@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_LICENSE_DIR@")
+hunter_test_string_not_empty("@HUNTER_GLOBAL_SCRIPT_DIR@")
+
+if (BUILD_SHARED_LIBS)
+  set(shared "static=no")
+else()
+  set(shared "static=yes")
+endif()
+
+# Use native slashes to help nmake if statements work
+file(TO_NATIVE_PATH "@HUNTER_PACKAGE_INSTALL_PREFIX@" XML_INSTALL_DIR)
+
+ExternalProject_Add(
+    "@HUNTER_EP_NAME@"
+    URL
+    "@HUNTER_PACKAGE_URL@"
+    URL_HASH
+    SHA1=@HUNTER_PACKAGE_SHA1@
+    DOWNLOAD_DIR
+    "@HUNTER_PACKAGE_DOWNLOAD_DIR@"
```

Comment:
Please add `TLS_VERIFY`:
* https://github.com/ruslo/hunter/blob/master/cmake/schemes/url_sha1_cmake.cmake.in#L132-L133